source/extensions/resource_monitors/downstream_connections/config.h

```diff
@@ -17,7 +17,7 @@ class ActiveDownstreamConnectionsMonitorFactory
               DownstreamConnectionsConfig> {
 public:
   ActiveDownstreamConnectionsMonitorFactory()
-      : ProactiveFactoryBase("envoy.resource_monitors.downstream_connections") {}
+      : ProactiveFactoryBase("envoy.resource_monitors.global_downstream_max_connections") {}
```

Comment:
Seems like the implementation was using `envoy.resource_monitors.downstream_connections`? Which the docs were also using?

Reply:
@KBaichoo the resources are being instantiated in overload manager impl code, e.g. [here](https://github.com/envoyproxy/envoy/blob/main/source/server/overload_manager_impl.cc#L439) we extract resource name and check if it is a proactive type or not and based on that invoke the corresponding factory method. Basically resource name is matched against a map with [this value](https://github.com/envoyproxy/envoy/blob/main/envoy/server/overload/thread_local_overload_state.h#L24) as key.